### DISCUSSION TITLE
Tive dificuldade na parte do Azure.

### DISCUSSION THREAD

**Discussion by Lcrmartins** [balta-io/7194#17] [Perguntas e Respostas] [answered]
Olá.
Estou fazendo este curso em setembro de 2021 e o Azure está um tanto diferente do mostrado no curso. Talvez necessite uma revisitada para atualização.
Aliado a isso. Foi apresentado muito rápido. Foram mostrados partes que acredito serem importantes no Azure, mas que não são de interesse no primeiro deploy e sendo uma conta gratuita.
Fui acompanhando na medida que conseguia me achar mas não consegui fazer o deploy.
A minha API está no Git mas não consegui fazê-la subir para o Azure. o log indica falha na build, code 145.
`Run dotnet build --configuration Release
A compatible installed .NET Core SDK for global.json version [3.0.100] from [/home/runner/work/balta-7194deploy/balta-7194deploy/global.json] was not found
  3.1.301 [/home/runner/.dotnet/sdk]
Install the [3.0.100] .NET Core SDK or update [/home/runner/work/balta-7194deploy/balta-7194deploy/global.json] with an installed .NET Core SDK:
Error: Process completed with exit code 145.`
Vou partir para o próximo curso e estudar o Azure e deploy em outra oportunidade.
Obrigado.
Luis

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @Lcrmartins estes vídeos vão te ajudar:

- https://www.youtube.com/watch?v=giCR4VgKjQY
- https://www.youtube.com/watch?v=cl2iaR7JyvE
